### DISCUSSION TITLE
New updates should only contain files that were modified

### DISCUSSION THREAD

**Discussion by takaji10** [TT-ReBORN/Georgia-ReBORN#15] [Q&A] [answered]
If a user wants to update their Georgia-ReBORN installation by downloading the latest package, all user-specific settings made by the user are wiped-out upon updating due to overwriting of `/configuration` files.

It could be as simple as choosing to not overwrite the files in this directory, but then the user is not sure if there was an important settings update in those files.

One suggestion could be to create a base package for this theme, and then future updates would only include files that have been modified.

This request comes from the fact that I just updated the theme after a bugfix, and some of the settings I made in the preferences window is now gone (ie. UI hacks main window settings).

**Comment by TT-ReBORN** [comment, marked as answer]
This not seem to be possible because I always need to update the configuration files when new user-component updates and changes in those user-components are made. I need to update those too. I personally found out when old config files are still present, it sometimes bugs foobar/SMP and also the user-components. It is recommended to always overwrite all files.

You could try the component foo_jesus:
https://wiki.hydrogenaud.io/index.php?title=Foobar2000:Components/Autosave_%26_Autobackup_(foo_jesus)
https://hydrogenaud.io/index.php/topic,68528.0.html

-TT

**Reply by takaji10** [reply]
Thanks, I'll give it a try!
I've also noticed that the actual number of overwritten settings is quite minimal, so perhaps it's not such a big issue.
